client/src/components/header/header.ts

```diff
@@ -0,0 +1,53 @@
+import styled, {keyframes} from "styled-components";
+import { Wrapper } from "../../box/flexBox";
+import {flipInX, swing} from 'react-animations'
+
+export const HeaderSection = styled.header`
+    background-color: rgba(0,0,0,0.65);
+    width: 100%;
+	height: 70px;

```

Comment:
It seems that it's time to set up Prettier, there's issue with formatting

Reply:
Saw it. I'll set up.

Reply:
It's all there straight now
i hsve just included Prettier yesterday